### PR TITLE
update for ida_script.py to handle ida 7.0

### PR DESCRIPTION
**Pull request by aidielse**

### CODE REVIEW

**Comment by disconnect3d** [comment]
What about `idaapi.save_database` - can you check that in IDA 7?

```
Python>help(idaapi.save_database)
Help on function save_database in module idaapi:

save_database(*args)
    save_database(outfile, delete_unpacked) -> bool
```

It seems to me this might work in all versions as I guess it is the idapython api and not this old(?) idc.

**Comment by disconnect3d** [comment]
@aidielse bump

**Comment by zachriggle** [comment]
LGTM

File: ida_script.py [hunk 1]
```diff
@@ -20,7 +20,10 @@
 dt = datetime.datetime.now().isoformat().replace(':', '-')
 
 # Save the database so nothing gets lost.
-idc.SaveBase(idc.GetIdbPath() + '.' + dt)
+if idaapi.IDA_SDK_VERSION == 700:
```
Comment (zachriggle) [inline ida_script.py]: Might be worth doing `>= 700`

**Review by zachriggle — CHANGES_REQUESTED** [review]

**Comment by aidielse** [comment]
Sorry I haven't responded, I've been pretty busy.
@disconnect3d that works in 7.0, updated.

**Comment by disconnect3d** [comment]
@aidielse Are you sure? I mean, I asked because this method is also present on IDA 6.x so we might omit having a condition for IDA version.

However it turns out it has two arguments:
```python
Traceback (most recent call last):
  File "C:\Program Files (x86)\IDA 6.8\python\idaapi.py", line 601, in IDAPython_ExecScript
    execfile(script, g)
  File "Z:/home/dc/installed/pwndbg/ida_script.py", line 25, in <module>
    idaapi.save_database(idc.GetIdbPath() + '.' + dt)
  File "C:\Program Files (x86)\IDA 6.8\python\idaapi.py", line 50712, in save_database
    return _idaapi.save_database(*args)
TypeError: save_database expected 2 arguments, got 1
```

So did they change it to have just one argument in IDA 7.x or did we miss something here?

**Comment by disconnect3d** [comment]
I spoke with @aidielse on IRC and he confirmed the `idaapi.save_database` works in IDA 7.x.

Thanks!

**Review by disconnect3d — APPROVED** [review]

**Pull request merged**